### PR TITLE
Feat/query entry

### LINKED ISSUE #6: [4.1.4] Admin/Normal User (Capstone Entry) - R: Query records
The interface shall allow the user to query the system to retrieve the information required.

The following are necessary for the search to start:
- Academic Year
- Keywords

The resultant of the query shall be displayed as a list of selectable values.

The values shall include
- Project Title
- Name of person

Selection of the item will result in the information of the project to be displayed.

Modifications to the information can then be done and shall require the user to save for the data to be properly modified by selecting “Modify”. Selecting “Return” shall return the user to the previous page. 

Only the Administrator account would be allowed to “Delete” a result.

Image

Image

Image

Image

### LINKED ISSUE #10: [4.3] Admin/Normal User (Capstone Entry) - U: Modify records

### LINKED ISSUE #11: [4.4] Admin (Capstone Entry) - D: Delete records
Image

### PR DESCRIPTION
**Pull request by Kuan-Zher**
Closes #6 , #10, #11

### CODE REVIEW

**Comment by jiachenyee** [comment]
Can you merge the main branch into your branch?

**Review by isabellepakyishan — APPROVED** [review]
lgtm! (i'm tired🫠)

**Comment by Kuan-Zher** [comment]
> Can you merge the main branch into your branch?

I already merged it

File: js/queryentryform.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+const recordAPIURL = "http://localhost:8001/api/records";
```
Comment (jiachenyee) [inline js/queryentryform.js]: Replace with global variable version

File: js/queryentrylist.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+const accountAPIURL = "http://localhost:8002/api/accounts";
+const recordAPIURL = "http://localhost:8001/api/records";
+
```
Comment (jiachenyee) [inline js/queryentrylist.js]: Replace these two with the global variable version

File: js/queryentrymodify.js [hunk 3]
```diff
@@ -0,0 +1,327 @@
+const accountAPIURL = "http://localhost:8002/api/accounts";
+const recordAPIURL = "http://localhost:8001/api/records";
+
```
Comment (jiachenyee) [inline js/queryentrymodify.js]: Replace with global variable

**Review by jiachenyee — CHANGES_REQUESTED** [review]
Can you merge the changes from Main into your branch and replace with the global variable

File: js/queryentryform.js (hunk 1 above)
Comment (Kuan-Zher) [reply js/queryentryform.js]: Changed

File: js/queryentrymodify.js (hunk 3 above)
Comment (Kuan-Zher) [reply js/queryentrymodify.js]: Changed

File: js/queryentrylist.js (hunk 2 above)
Comment (Kuan-Zher) [reply js/queryentrylist.js]: Changed

**Review by DevOps-2023-TeamA — APPROVED** [review]
lgtm!

**Pull request merged**